src/object.d

```diff
@@ -1974,6 +1975,11 @@ void clear(T : Value[Key], Value, Key)(T* aa)
     _aaClear(*cast(void **) aa);
 }
 
+void reserve(T : Value[Key], Value, Key)(ref T aa, size_t ndim)
```

Comment:
Could you please add an overload for T* as well?

Reply:
Done

Reply:
Ouch do we really need this `T*` for new overloads? After all aren't we trying to move away from the ugly C-like pointer syntax?
Also while you are at it, note that most AA methods still aren't `@safe` :/

Reply:
@wilzbach yes, it needs to be there. Otherwise it operates inconsistently with other normal objects that can call "members" based on a pointer.

This is a special case because AA's are builtins. If they were fully library types, this wouldn't be needed.

Reply:
sux but ok